addons/website/views/snippets/snippets.xml

```diff
@@ -1149,14 +1150,17 @@
                     <t t-set="data_fields">[&quot;code&quot;]</t>
                 </t>
             </t>
-            <t t-call="website.snippet_options_conditional_visibility">
-                <t t-set="option_name">⌙ Languages</t>
-                <t t-set="attribute_rule">visibilityValueLangRule</t>
-                <t t-set="save_attribute">visibilityValueLang</t>
-                <t t-set="attribute_name">lang</t>
-                <t t-set="model">res.lang</t>
-                <t t-set="call_with">code</t>
-                <t t-set="data_fields">[&quot;code&quot;]</t>
+            <t t-if="len(current_website.language_ids) > 1">
+                <t t-call="website.snippet_options_conditional_visibility">
+                    <t t-set="option_name">⌙ Languages</t>
+                    <t t-set="attribute_rule">visibilityValueLangRule</t>
+                    <t t-set="save_attribute">visibilityValueLang</t>
+                    <t t-set="attribute_name">lang</t>
+                    <t t-set="model">res.lang</t>
+                    <t t-set="call_with">code</t>
+                    <t t-set="domain">[["id", "in", <t t-out="current_website.language_ids.ids"/>]]</t>
```

Comment:
when you have technical content like this, please try to put it in `t-value` to avoid translating it (or put the block in a `t-translation="off"` to disable globally)

Reply:
Indeed :+1: or even `t-valuef` which might be better here, fix is coming

Reply:
Hi, I added a fix here https://github.com/odoo/odoo/pull/76219/commits/72a7614361031204c54dd25e6dd05eb71c867351

Reply:
thanks for the quick reply :+1: